src/transforms/merge-with.ts

```diff
@@ -14,6 +14,15 @@
 import { Transform, Observable } from "../types.js";
 import { merge } from "../combiners/merge.js";
 
+/**
+ * Merges another observable by emitting all items from the original observable
+ * then emitting all items from `other`.
```

Comment:
The "then" implies sequence, which is not the case for `merge`. The resulting observable emits the items that get emitted from _either_ observable.

Reply:
Thanks, I misunderstood how `merge` works.